openmc/statepoint.py

```diff
@@ -589,32 +629,30 @@ def get_tally(self, scores=[], filters=[], nuclides=[],
 
         Parameters
         ----------
-        scores : list
```

Comment:
There is actually a typo in my original comment above: "satisfieds" -> "satisfies"